inc/meta-box-html.php

```diff
@@ -178,6 +178,10 @@ private function draw_input( $field, $form_id = NULL ) {
 			HTML::text_area( $field['rows'], $field['cols'], $field['meta_key'], $value, $title, $label, $field['placeholder'], $required, $form_id );
 		}
 
+		if ( $field['type'] == 'wysiwyg' ) {
+			$this->wysiwyg( $value, $field['meta_key'], $field['params'], $label, $form_id );
+		}
```

Comment:
That's a good catch. In the PR #40, that is addressed. The way that labels are rendered for each field is cleaned up in it. Titles are only needed in cases when it makes sense to have a header tag instead of a label tag. For instance, where the field is either a "standalone" i.e. it is not inside a fieldset or formset. So this is okay as it stands.